server/src/test/scala/org/http4s/server/middleware/MaxActiveRequestsSpec.scala

```diff
@@ -0,0 +1,88 @@
+package org.http4s.server.middleware
+
+import cats.implicits._
+import cats.effect.IO
+import cats.data._
+import cats.effect.concurrent._
+import org.http4s._
+import org.specs2.execute.{AsResult, Failure, Result}
+import scala.concurrent.duration._
+
+class MaxActiveRequestsSpec extends Http4sSpec {
+
+  val req = Request[IO]()
+
+  protected val Timeout = 10.seconds
+
+  def routes(deferred: Deferred[IO, Unit]) = Kleisli { req: Request[IO] =>
```

Comment:
This is missing the changes in #2788

Reply:
Right, that hadn't been merged to 0.20 yet. It will catch the next train.